.github/wiki/Testing.md

```diff
@@ -0,0 +1,127 @@
+NimbleTemplate uses Github Action as the CI, the workflow files locate under [.github/workflows/](https://github.com/nimblehq/elixir-templates/tree/develop/.github/workflows) directory.
+
+There are 2 types of test **Template test** and **Variant test**
+
+
+### 1/ Template test
+
+All test files are located under `test/` directory.
+
+```
+.
+├── ...
+├── test
+│   ├── ...
+│   ├── nimble_template
+│   │   └── addons
+│   │   │   ├── ...
+│   │   │   ├── common_addon_test.exs
+│   │   │   └── variants
+│   │   │   │   └── mix
+│   │   │   │   │   ├── ...
+│   │   │   │   │   └── mix_addon_test.exs
+│   │   │   │   └── phoenix
+│   │   │   │   │   └── api
+│   │   │   │   │   │   ├── ...
+│   │   │   │   │   │   └── api_addon_test.exs
+│   │   │   │   │   └── live
+│   │   │   │   │   │   ├── ...
+│   │   │   │   │   │   └── live_addon_test.exs
+│   │   │   │   │   └── web
+│   │   │   │   │   │   ├── ...
+│   │   │   │   │   │   └── web_addon_test.exs
+```
+
+### 2/ Variant test
+
+#### 2.1/ Variant
```

Comment:
I think we can remove this heading as we only have 2 _major_ variants, being mix and phoenix 🤔 
```suggestion
```